### DISCUSSION TITLE
Why don't we just use a ctor in HttpExceptionMiddleware?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [ASP-NET-MVC-Boilerplate/Framework#63] [Q&A]
https://github.com/Dotnet-Boxed/Framework/blob/b2a0638db85d3ed48d94a6818160bbcc7385f798/Source/Boxed.AspNetCore/Middleware/HttpExceptionMiddleware.cs#L20

**Comment by RehanSaeed** [comment]
I have no idea why I did that.....

Now that I think about it though. The exception case is rare, so by not using the constructor, we only create a logger in the rare case that an exception is thrown. So it works out in the end.
